### PR TITLE
Refactor into modular mapper models mmmmm....

### PR DESCRIPTION
**Pull request by lostphilosopher**
- Making Attendee, Event, and AttendeeEvent accessible as objects obtained from the API.  
- All methods are basic mirrors of Rails routes (index, show, update, create).
- There are VCR powered unit tests for each method.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# These vars are only good for testing
+GARAGE_CALENDAR_API_USER_ID=6f5c9d43-c0e8-4af7-9971-bbe39a3a8e68
+GARAGE_CALENDAR_API_USER_EMAIL=garagecalendarapigem@example.com
+GARAGE_CALENDAR_API_USER_TOKEN=thegarage
```
Comment (dbelling) [inline .env]: I've been using `.env.local` for testing on my end, and storing as appropriate on Heroku on Travis. This should be fine for gem testing, just worried about checking in ENV variables to one of our few public repos.

File: lib/garage_calendar_api/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
-class GarageCalendarApi
+module GarageCalendarApi
   VERSION = "0.0.1"
```
Comment (dbelling) [inline lib/garage_calendar_api/version.rb]: Thoughts on versioning the gem to mirror the API? `VERSION == 1.0.0`

File: spec/garage_calendar_api/attendee_event_spec.rb [hunk 3]
```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+describe GarageCalendarApi::AttendeeEvent do
+  let(:id) { 'fc3460d0-1ea9-419e-8404-67c530b66ae4' }
```
Comment (dbelling) [inline spec/garage_calendar_api/attendee_event_spec.rb]: Maybe a place for an environment variable, just to DRY things up a bit? 
See also, `attendee_spec.rb` and `event_spec.rb`

**Comment by dbelling** [comment]
Just a few comments, otherwise -- LGTM :+1:

File: .env (hunk 1 above)
Comment (lostphilosopher) [inline .env]: The idea is that someone should be able to clone the gem and run the tests themselves. To do that (in the current world order) they'll need API credentials and certain records to be present in the database and associated with those records. Maybe a staging API environment is the next step to take.

File: lib/garage_calendar_api/version.rb (hunk 2 above)
Comment (lostphilosopher) [inline lib/garage_calendar_api/version.rb]: AH! Yes. Good call.

**Pull request merged**